.github/workflows/linters.yml

```diff
@@ -46,6 +46,5 @@ jobs:
           components: clippy
       - uses: Swatinem/rust-cache@v2
       - uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --workspace --all-features --all-targets -- -D warnings
+      - run: >
+          scripts/clippy.sh
```

Comment:
why not add the args present in the script in the args key?

Reply:
since we need the exact same command in githooks as well i was trying to reduce duplication, but will revert back to using what you suggested